topic-operator/src/main/resources/log4j2.properties

```diff
@@ -9,3 +9,13 @@ rootLogger.level = ${env:STRIMZI_LOG_LEVEL:-INFO}
 rootLogger.appenderRefs = stdout
 rootLogger.appenderRef.console.ref = STDOUT
 rootLogger.additivity = false
+
+## Kafka AdminClient logging is a bit noisy at INFO level
+#logger.kafka.name = org.apache.kafka
+#logger.kafka.level = ${env:STRIMZI_AC_LOG_LEVEL:-WARN}
+#logger.kafka.additivity = false
+#
+## Zookeeper is very verbose on INFO level , set it to WARN by default
+#logger.zookeepertrustmanager.name = org.apache.zookeeper
+#logger.zookeepertrustmanager.level = ${env:STRIMZI_ZOOKEEPER_LOG_LEVEL:-WARN}
+#logger.zookeepertrustmanager.additivity = false
```

Comment:
Why do we have the commented out code here? Its not like anyone can uncomment it when it is baked into the JARs.